vue-frontend/src/components/portfolio/BrandingSection.vue

```diff
@@ -169,9 +169,9 @@
           <div
             class="v2-normal-text tw-bg-white tw-font-light ..."
             :class="[
-              data.title
+              data.title !== 'togness'
                 ? 'tw-px-6 tw-py-12 sm:tw-px-8 lg:tw-px-12 lg:tw-py-12 xl:tw-px-20 xl:tw-py-20 tw-text-center'
-                : 'tw-mt-12',
+                : 'tw-hidden',
```

Comment:
Had you seen difference between UI of togness portfolio in live and local?

![Screenshot from 2023-03-27 12-28-50](https://user-images.githubusercontent.com/69897605/227864408-21806360-d045-4cf4-b43c-891f53875d0f.png)

![Screenshot from 2023-03-27 12-29-09](https://user-images.githubusercontent.com/69897605/227864392-1a6b73cb-d1ff-49a7-a612-cb511c337a07.png)